tool/gravity/cli/commands.go

```diff
@@ -442,6 +442,18 @@ type AutoJoinCmd struct {
 	SystemDevice *string
 	// Mounts is additional app mounts
 	Mounts *configure.KeyVal
+	// PeerAddr specifies the address of the cluster node to join
```

Comment:
Nit: This wording is a bit ambiguous, does it imply "address of the node to join **to**" or "address of the joining node"?